draft-ietf-cbor-7049bis.md

```diff
@@ -437,11 +442,11 @@ computed from the argument); in major type 2 and 3 it gives the length
 of the string data in bytes that follows; and in major types 4 and 5 it is used to
 determine the number of data items enclosed.
 
-If the encoded sequence of bytes ends before the end of a data item
-would be reached, that encoding is not well-formed. If the encoded
+If the encoded sequence of bytes ends before the end of a data item,
+that item is not well-formed. If the encoded
 sequence of bytes still has bytes remaining
-after the outermost encoded item is decoded, that encoding is not a
-single well-formed CBOR item.
+after the outermost encoded item is decoded, that outer item is
+not well-formed.
```

Comment:
CBOR is self-delimiting, and some protocols use that.  So the decoder may not now where the boundary is — it is actually its job to find out.  I think we need to expose this concept somewhere and just refer to it here.

Reply:
How's this phrasing? I don't want this PR to embark on adding a section about using decoders to find the end of an item.